src/timed-stream.ts

```diff
@@ -42,11 +43,11 @@ class StreamTimer {
   }
 
   start() {
-    this.startTime = process.hrtime.bigint();
+    this.startTime = hrtime.bigint();
```

Comment:
This more closely matches the way hrtime is used elsewhere and is easier to mock in tests.